requirements.txt

```diff
@@ -1,2 +1,2 @@
 clickhouse-driver~=0.2.0
-apache-airflow>=2.0.0,<2.3.0
+apache-airflow>=2.0.0,<=2.3.*
```

Comment:
Is it a correct specification? Here is what have been installed by CI/CD: https://github.com/whisklabs/airflow-clickhouse-plugin/runs/6510277274?check_suite_focus=true#step:4:14

```
Collecting apache-airflow<=2.3.*,>=2.0.0
  Downloading apache_airflow-2.2.5-py3-none-any.whl (5.3 MB)
```

Maybe smth like `apache-airflow>=2.0.0,~=2.3.0` is a proper one? Not sure if asterisk works.